From vijaykumar@bravegnu.org  Wed Oct 29 10:48:28 2008
From: Vijay Kumar <vijaykumar@bravegnu.org>
Date: Wed, 29 Oct 2008 08:58:36 +0530
Subject: Staging: poch: Fix build warnings
To: greg@kroah.com
Cc: jayakumar.lkml@gmail.com, alexey.zaytsev@gmail.com, robfitz@273k.net, bdonnette@linagora.com, htejun@gmail.com, <abixby@redrapids.com>
Message-ID: <20081029033332.718653466@bravegnu.org>


Removed out printing of DMA address, that causes warnings during
build.

Signed-off-by: Vijay Kumar <vijaykumar@bravegnu.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/poch/poch.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/staging/poch/poch.c
+++ b/drivers/staging/poch/poch.c
@@ -390,8 +390,8 @@ static int poch_channel_alloc_groups(str
 		group->user_offset =
 			(header_pages + (i * group_pages)) * PAGE_SIZE;
 
-		printk(KERN_INFO PFX "%ld: user_offset: 0x%lx dma: 0x%x\n", i,
-		       group->user_offset, group->dma_addr);
+		printk(KERN_INFO PFX "%ld: user_offset: 0x%lx\n", i,
+		       group->user_offset);
 	}
 
 	return 0;
